backend/ponder.schema.ts

```diff
@@ -217,6 +218,28 @@ export const bridgeTransaction = onchainTable(
   })
 )
 
+/**
+ * Track vault share price snapshots over time
+ * - id: Composite primary key in format vault-blockNumber
+ * - vault: Vault (pool) contract address
+ * - blockNumber: Block number when the snapshot was taken
+ * - timestamp: Block timestamp when the snapshot was taken
+ * - sharePrice: Share price at snapshot time, computed via convertToAssets(1e18)
+ */
+export const vaultsnapshot = onchainTable(
+  'vaultsnapshot',
+  (t) => ({
+    id: t.text().primaryKey(),
+    vault: t.hex().notNull(),
```

Comment:
We need the network in here as the vault address is not unique across all networtks!